spec/spec_helper.rb

```diff
@@ -11,12 +11,24 @@
 # a separate helper file that requires the additional dependencies and performs
 # the additional setup, and require it from the spec files that actually need
 # it.
-#
+
+require 'active_fedora/cleaner'
+
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
 RSpec.configure do |config|
   # rspec-expectations config goes here. You can use an alternate
   # assertion/expectation library such as wrong or the stdlib/minitest
   # assertions if you prefer.
+
+  config.before(:suite) do
+    ActiveFedora::Cleaner.clean!
```

Comment:
@val99erie we've run into problems when we clean without re-creating the admin set on other projects (mahonia).